code/modules/mob/living/basic/space_fauna/netherworld/migo.dm

```diff
@@ -36,10 +36,11 @@
-	if(prob(0.04)) //1 in 20 chance on-load mi-gos will spawn with a miku wig on.
-		base_icon_state = "mi-go-h"
-		name = "hatsune mi-go"
-		desc += " This one is wearing a bright blue wig."
+
+	if(!istype(src, /mob/living/basic/migo/hatsune) && prob(0.1)) // chance on-load mi-gos will spawn with a miku wig on (shiny variant)
```

Comment:
Won't this possibly but albeit very, very rarely cause issues with the poly belt? I mean, the 0.1% chance is a pretty much unfathomably low, but still...

Reply:
there's probably more mobs that this would be an issue with when you think about it. that would be addressed in a separate PR. if you aren't comfortable with merging it without that PR being opened I can look into it

Reply:
Ok so, just revert it back to 1 in 200.

Reply:
should maybe mention this in a comment, then

Reply:
> wouldn't an alternate way to do this just be to just retain the old way this was done and make hatsune mi-gos not roll this chance?

That's not an option because it's the wrong way to do it. Look closer at the old code and my breakdown in the PR body and you'll see the flaws. It gets exponentially worse if people want to do cuter things (like give the hatsune migo a different soundtrack, etc.)

We shouldn't sacrifice the right way to do things to cover up a failure in polymorph belt code (the fact that it could qdel you if you switch into a mob that gets del'd on spawn). Something similar could already be extant in stuff like those lavaland killworms evolution